syft/execution/communication.py

```diff
@@ -28,6 +27,14 @@ def __init__(
         super().__init__()
 
         self.obj_id = obj_id
+        if name in ["move", "remote_send", "mid_get", "remote_get", "get", "share", "share_"]:
```

Comment:
We can probably use this as the list of supported methods to determine whether a particular action is a computation or a communication in the tracing. Could the list move to a (hard-coded) static method?

Reply:
can do!